src/main/java/com/avispl/symphony/dal/communicator/management/zoom/rooms/ZoomRoomsCommunicator.java

```diff
@@ -0,0 +1,581 @@
+            }
+        });
+    }
+
+    /**
+     * Get map of parameters describing the currently connected cameras,
+     * including information like selected/non-selected, name, id, alias etc
+     *
+     * @return {@link Map} containing parameters mapped out of the string feed like:
+     *
+     *      *s Video Camera Line 1 id: 00#8&2cc2822b&0&0000#{65e8773d-8f56-11d0-a3b9-00a0c9223196}\global
+     *      *s Video Camera Line 1 Name: Logi Rally Camera
+     *      ......
+     *      *s Video Camera Line 2 combinedDevice: off
+     *      *s Video Camera Line 2 numberOfCombinedDevices: 0
+     *      *s Video Camera Line 2 ptzComId: -1
+     *
+     * @throws Exception during ssh communication
+     */
+    private Map<String, String> retrieveCameraLineParameters() throws Exception {
+        String cameraLine = execute(ZSTATUS_CAMERA_LINE, "*s Video Camera Line");
+        return parseZoomRoomsProperties(cameraLine, "*s Video Camera Line");
+    }
+
+    /**
+     * Parse properties parameters based on the provided start of the line,
+     * so the expected format of data provided here is:
+     * Video Camera Line 1 Name: Camera Name
+     * .....
+     * Video Camera Line 2 Name: Camera Name
+     *
+     * @param response           string from Zoom Rooms CLI
+     * @param propertyIdentifier supposed start of the string
+     * @return map of a format <Video Camera Line 1 Name: Camera Name, Video Camera Line 2 Name: Camera Name....>
+     */
+    private Map<String, String> parseZoomRoomsProperties(String response, String propertyIdentifier) {
+        Map<String, String> parameters = new HashMap<>();
+        Arrays.asList(response.split("\n")).forEach(s -> {
+            if (s.startsWith(propertyIdentifier)) {
+                String[] values = s.split(":");
+                parameters.put(values[0].replace("*s ", ""), values[1].replace("\r", "").trim());
+            }
+        });
+        return parameters;
+    }
+
+    /**
+     * Join meeting that is currently in progress or start a scheduled one.
+     *
+     * @param value - contains dialString of a format %meetingNumber%.%meetingPassword%@zoomcrc.com
+     *              password is optional.
+     * @return meetingId if operation is successful
+     */
+    private String joinStartMeeting(String value) throws Exception {
+        boolean hasPassword;
+        String meetingNumber;
+        String meetingPassword;
+
+        Pattern dialStringPattern = Pattern.compile("(\\d+)\\.?(\\d+)?(@[a-z]+?\\.[a-z]{2,5})");
+        Matcher matcher = dialStringPattern.matcher(value);
+        if (!matcher.find() && !matcher.matches()) {
+            // This might be the section to call out SIP/h323 devices it "invite" functionality fits our needs
+            // However, for the simplified approach, this option is skipped for now
+            throw new OperationNotSupportedException("Dial string does not match expected pattern meetingNumber.meetingPassword@zoomcrc.com");

```

Comment:
Does not look like correct exception. Operation is supported, the passed parameter is wrong. Should be IllegalParameterException or similar probably.
Also the error message might not match to what the pattern above is doing.